android/java/org/chromium/chrome/browser/toolbar/top/BraveToolbarLayoutImpl.java

```diff
@@ -766,7 +777,7 @@ public void onClick(View v) {
                     getContext().getResources().getString(R.string.already_added_in_playlist),
                     snackBarActionModel);
         } catch (BraveActivity.BraveActivityNotFoundException e) {
-            Log.e(TAG, "showAddedToPlaylistSnackBar " + e);
+            Log.e(TAG, "showAlreadyAddedToPlaylistSnackBar " + e);
```

Comment:
It's best to include the exception as a parameter. Doing so, it will be included in the stacktrace without trunkation.

```suggestion
            Log.e(TAG, "showAlreadyAddedToPlaylistSnackBar", e);
```